homeassistant/components/almond/strings.json

```
@@ -8,7 +8,7 @@
       }
     },
     "abort": {
-      "already_setup": "You can only configure one Almond account.",
+      "already_setup": "[%key:common::config_flow::abort::single_instance_allowed%].",
       "cannot_connect": "Unable to connect to the Almond server.",
```

Please replace this value as well.
```suggestion
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
```